site/content/docs/5.1/layout/containers.md

```diff
@@ -151,22 +154,19 @@ Use `.container-fluid` for a full width container, spanning the entire width of
 
 ## Sass
 
+### Variables
+
 As shown above, Boosted generates a series of predefined container classes to help you build the layouts you desire. You may customize these predefined container classes by modifying the Sass map (found in `_variables.scss`) that powers them:
 
 {{< scss-docs name="container-max-widths" file="scss/_variables.scss" >}}
 
+### Mixins
+
 In addition to customizing the Sass, you can also create your own containers with our Sass mixin.
 
-```scss
-// Source mixin
-@mixin make-container($padding-x: $container-padding-x) {
-  width: 100%;
-  padding-right: $padding-x;
-  padding-left: $padding-x;
-  margin-right: auto;
-  margin-left: auto;
-}
+{{< scss-docs name="container-mixins" file="scss/mixins/_container.scss" >}}
```

Comment:
If their `@mixin make-container` example is supposed to be the same the code than in `scss/mixins/_container.scss`, then you could propose your modification to Bootstrap. 

Because what's displayed here is different from https://github.com/twbs/bootstrap/blob/main/scss/mixins/_container.scss.